plenum/server/consensus/ordering_service.py

```diff
@@ -1600,6 +1602,8 @@ def _can_order(self, commit: Commit) -> Tuple[bool, Optional[str]]:
 
         key = (commit.viewNo, commit.ppSeqNo)
         if self._validator.has_already_ordered(*key):
+            if self._data.prev_view_prepare_cert == commit.ppSeqNo:
```

Comment:
It's not good to do some business logic (sending messages) in check (_can_do_something) methods